src/rulesets/rest/2022-05-25/json-api-rules/__tests__/resource-object-rules.test.ts

```diff
@@ -129,7 +129,7 @@ describe("resource object rules", () => {
     );
 
     test.each(["uuid", "uri", "ulid"])(
-      "passes when PATCH request body for a relationship is of the correct form (data is an collection of resource objects with id and type)",
+      "passes when PATCH request body for a relationship is of the correct form (data is a single resource object with id and type)",
```

Comment:
We still want a test for collections to make sure we don't regress

Reply:
oops... I probably replaced it rather than adding! Thanks!